analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -465,6 +470,37 @@ def get_query(source):
                                        "Jet_jetId": h.Jet_jetId,
                                       })
 
+def get_uproot_raw_query():
+    cut = '((count_nonzero((Electron_pt > 30) & (abs(Electron_eta) < 2.1) & (Electron_cutBased == 4) & (Electron_sip3d < 4), axis=1)' \
```

Comment:
Is there a good way to write this in a way that is not a string for syntax highlighting purposes? I was hoping one could write this as a function taking a tree and returning an awkward array, allowing to also easily debug this locally by feeding a tree from uproot.

Reply:
Unfortunately this is not at all trivial to do (if you really want to do that what you want is the Python codegen instead). In some later release we can add a capability to reasonably easily test it locally with `uproot` calls. (It's not trivial at the moment because we have to manually bootstrap awkward functions into the uproot execution space.)

At least JupyterLab does highlighting of the parentheses in the strings, so it's actually not so terrible to write the expressions correctly.